### PR TITLE
Add changelog for patch release 19.4

### PR DESCRIPTION
**Pull request by rahulbabu95**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -30,6 +30,32 @@ description: >
...
+  - `v1-25-eks-35` to [`v1-25-eks-37`](https://distro.eks.amazonaws.com/releases/1-25/37/)
+  - `v1-26-eks-31` to [`v1-26-eks-33`](https://distro.eks.amazonaws.com/releases/1-26/33/)
+  - `v1-27-eks-25` to [`v1-27-eks-27`](https://distro.eks.amazonaws.com/releases/1-27/27/)
+  - `v1-28-eks-18` to [`v1-28-eks-20`](https://distro.eks.amazonaws.com/releases/1-28/20/)
+  - `v1-29-eks-7` to [`v1-29-eks-9`](https://distro.eks.amazonaws.com/releases/1-29/9ßßß/)
+
+### Fixed
+- Added processor for Tinkerbell Template Config [#7980](https://github.com/aws/eks-anywhere/pull/7980)
+- Added nil check for eksa-version when setting etcd url [#8018](https://github.com/aws/eks-anywhere/pull/8018)
+- Fixed registry mirror secret credentials set to emptry [#7933](https://github.com/aws/eks-anywhere/pull/7933)
```
Comment (abhay-krishna) [inline docs/content/en/docs/whatsnew/changelog.md:57]: nit:
```suggestion
- Fixed registry mirror secret credentials set to empty [#7933](https://github.com/aws/eks-anywhere/pull/7933)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -30,6 +30,32 @@ description: >
...
+
+### Changed
+- Support Docs site for penultime EKS-A version [#8010](https://github.com/aws/eks-anywhere/pull/8010)
+- Update Ubuntu 22.04 ISO URLs to latest stable release [#3114](https://github.com/aws/eks-anywhere-build-tooling/pull/3114)
+- Upgraded EKS-D:
+  - `v1-25-eks-35` to [`v1-25-eks-37`](https://distro.eks.amazonaws.com/releases/1-25/37/)
+  - `v1-26-eks-31` to [`v1-26-eks-33`](https://distro.eks.amazonaws.com/releases/1-26/33/)
+  - `v1-27-eks-25` to [`v1-27-eks-27`](https://distro.eks.amazonaws.com/releases/1-27/27/)
+  - `v1-28-eks-18` to [`v1-28-eks-20`](https://distro.eks.amazonaws.com/releases/1-28/20/)
+  - `v1-29-eks-7` to [`v1-29-eks-9`](https://distro.eks.amazonaws.com/releases/1-29/9ßßß/)
```
Comment (abhay-krishna) [inline docs/content/en/docs/whatsnew/changelog.md:52]: There seem to be some extra special characters here
```suggestion
  - `v1-29-eks-7` to [`v1-29-eks-9`](https://distro.eks.amazonaws.com/releases/1-29/9/)
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -30,6 +30,32 @@ description: >
...
+- Update Ubuntu 22.04 ISO URLs to latest stable release [#3114](https://github.com/aws/eks-anywhere-build-tooling/pull/3114)
+- Upgraded EKS-D:
+  - `v1-25-eks-35` to [`v1-25-eks-37`](https://distro.eks.amazonaws.com/releases/1-25/37/)
+  - `v1-26-eks-31` to [`v1-26-eks-33`](https://distro.eks.amazonaws.com/releases/1-26/33/)
+  - `v1-27-eks-25` to [`v1-27-eks-27`](https://distro.eks.amazonaws.com/releases/1-27/27/)
+  - `v1-28-eks-18` to [`v1-28-eks-20`](https://distro.eks.amazonaws.com/releases/1-28/20/)
+  - `v1-29-eks-7` to [`v1-29-eks-9`](https://distro.eks.amazonaws.com/releases/1-29/9/)
+
+### Fixed
+- Added processor for Tinkerbell Template Config [#7980](https://github.com/aws/eks-anywhere/pull/7980)
```
Comment (vivek-koppuru) [inline docs/content/en/docs/whatsnew/changelog.md:55]: This is the only one that has an issue, lets link the issue instead unless this only solves part of the issue

**Review by vivek-koppuru — COMMENTED** [review]
Also we are merging this to release branch first before main?

File: docs/content/en/docs/whatsnew/changelog.md (hunk 2 above)
Comment (vivek-koppuru) [reply docs/content/en/docs/whatsnew/changelog.md]: @abhay-krishna That was our secret version 🤣

File: docs/content/en/docs/whatsnew/changelog.md (hunk 3 above)
Comment (rahulbabu95) [reply docs/content/en/docs/whatsnew/changelog.md:55]: Good catch.

**Comment by rahulbabu95** [comment]
Created another PR to merge to main first.

**Pull request closed without merging**